src/commands/module/add.ts

```diff
@@ -15,6 +15,12 @@ import {
 } from './_utils'
 import type { NuxtModule } from './_utils'
 
+// Maybe we can import the list from somewhere, but idk
```

Comment:
in general I would think modules should match `nuxt` rather than forcing modules to development mode. they will be required in the same environment as nuxt (i.e the build environment).

if anything we could support a flag to allow the user to choose (`--dev`).

Reply:
(we could also add this list to the module registry meta btw)